pkg/lib/registry/registry.go

```diff
@@ -43,14 +43,15 @@ func (r RegistryUpdater) AddToRegistry(request AddToRegistryRequest) error {
 		if err := loader.Populate(); err != nil {
 			err = fmt.Errorf("error loading bundle from image: %s", err)
 			if !request.Permissive {
-				r.Logger.WithError(err).Fatal("permissive mode disabled")
+				r.Logger.WithError(err).Error("permissive mode disabled")
```

Comment:
since this isn't exiting anymore, is where do we actually check the error and fail if we're not `permissive`?

Reply:
[Here](https://github.com/operator-framework/operator-registry/blob/727f39a7bb54721a11bd026193618c31547a8c27/cmd/opm/registry/add.go#L68-L71) and [here](https://github.com/operator-framework/operator-registry/blob/727f39a7bb54721a11bd026193618c31547a8c27/pkg/lib/indexer/indexer.go#L96-L99)->[here](https://github.com/operator-framework/operator-registry/blob/727f39a7bb54721a11bd026193618c31547a8c27/cmd/opm/index/add.go#L126-L129).